en.js

```diff
@@ -459,9 +459,9 @@ module.exports = {
             }
         },
         objectives_field_records_title: "Records",
-        objectives_field_records_prefix: "Record de {statUnit}",
-        objectives_field_objective_recval: "Record de `{recval}` {statUnit} {timerange}",
-        objectives_field_objective_timestamp: "Record atteint le {timestamp}",
+        objectives_field_records_prefix: "{statUnit} record",
+        objectives_field_objective_recval: "`{recval}` {statUnit} {timerange} record",
```

Comment:
C'est mal exprimé ici, mais il s'agit plutot de remplacer selon les valeurs suivantes : 

1. {recval} = record value, donc la valeur du record atteint (int)
2. {statUnit} = unité statistique, par exemple "messages" ou "membres" dans "Record de 500 **messages**"
3. {timerange} = plage temporelle, par exemple "Record de 30 messages **en 24h**"

Donc là cela ferait par exemple : "20 messages in 24h record", et ça m'a l'air d'être une tournure un peu étrange, avec de la verbose et qui fait un peu rafistolé *(du moins je crois)*.
Donc ptêtre inverser certains termes et valeurs pour rembourrer la phrase, ou pas, à voir :)

Reply:
j'ai changé en "Record of `{recval}` {statUnit} {timerange}", c'est mieux?